pkg/operator/sync.go

```diff
@@ -927,6 +928,61 @@ func (optr *Operator) safetySyncControllerConfig(config *renderConfig) error {
 // the operator version and controller version match. We can call it from safetySyncControllerConfig
 // because safetySyncControllerConfig ensures that the operator and controller versions match before it syncs.
 func (optr *Operator) syncControllerConfig(config *renderConfig) error {
+	editCCAnno := false
+	kubeConfigData, err := optr.clusterCmLister.ConfigMaps("openshift-config-managed").Get("kube-apiserver-server-ca")
+	if err != nil {
+		klog.Errorf("Could not get kube-apiserver-server-ca, %v", err)
+	} else {
+		data, err := cmToData(kubeConfigData, "ca-bundle.crt")
+		if err != nil {
+			klog.Errorf("kube-apiserver-server-ca ConfigMap not populated yet. %v", err)
+		} else if data != nil {
+			binData := make(map[string][]byte)
+			binData["ca-bundle.crt"] = data
+			cmNew := corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "kubeconfig-data",
+				},
+				BinaryData: binData,
+			}
+			cm, getErr := optr.kubeClient.CoreV1().ConfigMaps("openshift-machine-config-operator").Get(context.TODO(), "kubeconfig-data", metav1.GetOptions{})
+			if err != nil && !errors.IsNotFound(getErr) {
+				klog.Errorf("Issue getting the kubeconfig-data configmap: %v", getErr)
+				return getErr
+			}
+			dataOld, err := cmToData(kubeConfigData, "ca-bundle.crt")
+			if err != nil && getErr == nil {
+				klog.Errorf("Could not get ca-bundle.crt from kubeconfig-data cm %v", err)
```

Comment:
question: If we encounter an error from `cmToData()`, what should happen? Should we continue or should we return that error from here?

Reply:
we should continue. I don't think not having this data on one sync should cause the whole operator to error. Might just be a timing fluke